rundeckapp/src/test/groovy/org/rundeck/app/providers/GormUserDataProviderSpec.groovy

```diff
@@ -74,7 +105,7 @@ class GormUserDataProviderSpec extends RundeckHibernateSpec implements DataTest
     def "Should throw error on registerLogin with bad login"() {
         setup:
         provider.configurationService = Mock(ConfigurationService) {
-            1 * getBoolean(UserService.SESSION_ID_ENABLED, false) >> false
```

Comment:
why is this being removed?

Reply:
After removing getUserByLoginOrCreate for findOrCreateUser this method was never called an the test was failing.